webui/react/src/pages/ResourcePool/Topology.tsx

```diff
@@ -59,6 +59,19 @@ const NodeElement: React.FC<PropsWithChildren<NodeElementProps>> = ({ name, slot
 };
 
 const Topology: React.FC<PropsWithChildren<Props>> = ({ nodes }) => {
+  const MAX_USABLE_NODES = 100;
+  if (nodes.length > MAX_USABLE_NODES) {
```

Comment:
we want to base this off of both max nodes max slots
how can we get access to max slots?

Reply:
yes. should be fixed

Reply:
@keita-determined did we agree on 1,000 nodes instead of 100?

Reply:
Yes. the latest code condition is `nodes > 1,000 OR slots > 10,000`